distribution.pm

```diff
@@ -114,9 +114,11 @@ Deprecated mode
 
   script_run($program, [$timeout])
 
-Run I<$cmd> (by assuming the console prompt and typing the command). After that, echo
-hashed command to serial line and wait for it in order to detect execution is finished.
-To avoid waiting, use I<$timeout> 0.
+Run I<$cmd> (by assuming the console prompt and typing the command). After
+that, echo hashed command to serial line and wait for it in order to detect
+execution is finished. To avoid waiting, use I<$timeout> 0. If a terminator
+is found, for example '&', C<script_run> tries to produce a valid bash
+command line but raises an error about the not allowed use.
```

Comment:
This still seems a bit weird. The script tries to produce a valid command line and *at the same time* raises an error about it. Judging by the code the function actually only raises an error. I would just write "The command must not be terminated with '&' (or an exception is thrown).".

Reply:
Yes, you are right. Fixed